Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+pipeline {
+        agent any
+        environment {
+      	           GIT_HASH = GIT_COMMIT.take(7)
+        }
+        stages{
+                   stage('Install dependencies') {
+                         steps {
+                              sh 'npm i'
+		              echo 'Building..'
+                              echo "Running job: ${env.JOB_NAME}\n Build: ${env.BUILD_ID} - ${env.BUILD_URL}\nPepeline: ${env.RUN_DISPLAY_URL}"
+                         }
+                   }   
+                    stage('Build') {
+                          steps {
+                               sh 'npm run build'
+                         }
+                   }
+                    stage('Export') {
+                          steps {
+                               sh 'npm run export'
+                          }
+                   }
+                    stage('Version Number') {
+                          steps {
+		               sh 'echo Version Number: "$GIT_HASH"';
```

Comment:
usually git_hush is not enough. Could you please use like 1.1.jenkins_build_number-git_hash-datetime